sbi/inference/posteriors/direct_posterior.py

```diff
@@ -261,6 +263,8 @@ def sample(
                 will draw init locations from prior, whereas `sir` will use Sequential-
                 Importance-Resampling using `init_strategy_num_candidates` to find init
                 locations.
+            return_device: device of the returned samples. Default to CPU because
+                usually only the evaluation and sampling happens on the GPU.
```

Comment:
Not sure if we really need this (it seems like a corner-case which the user could also take care of themselves). If adds another argument to user-facing `.sample()`.

If you think it's useful I'm also fine with keeping it.

Reply:
Yes, I am not sure about this either. I will try to avoid it.

Reply:
So, we decided to keep this?